Author: Muammar El Khatib <muammar@debian.org>
Description: This patch make possible to use python2.7 instead of python2.6.
Bug-Debian: http://bugs.debian.org/678023
Last-Update: 2012-07-01
Index: makehuman-1.0.0~alpha6/compile_src.sh
===================================================================
--- makehuman-1.0.0~alpha6.orig/compile_src.sh	2011-03-01 02:24:41.000000000 +0100
+++ makehuman-1.0.0~alpha6/compile_src.sh	2012-06-27 01:29:45.619478249 +0200
@@ -1 +1 @@
-gcc -g -o makehuman ./src/main.c ./src/glmodule.c ./src/core.c ./src/arraybuffer.c /usr/src/glew/glew.c -I/usr/include/SDL/ -I/usr/include/python2.6/ -I/usr/X11R6/include -I./include -DGLEW_STATIC -lpython2.6 -lX11 -lGL -lGLU -lSDL -Wall
+gcc -g -o makehuman ./src/main.c ./src/glmodule.c ./src/core.c ./src/arraybuffer.c /usr/src/glew/glew.c -I/usr/include/SDL/ -I/usr/include/python2.7/ -I/usr/X11R6/include -I./include -DGLEW_STATIC -lpython2.7 -lX11 -lGL -lGLU -lSDL -Wall
